miner.c

```
@@ -2663,12 +2667,14 @@ static bool jobj_binary(const json_t *obj, const char *key,
 	tmp = json_object_get(obj, key);
 	if (unlikely(!tmp)) {
 		if (unlikely(required))
-			applog(LOG_ERR, "JSON key '%s' not found", key);
+			if (opt_morenotices)
+				applog(LOG_ERR, "JSON key '%s' not found", key);
```

I see no reason to be hiding errors...